crashlogger.js

```diff
@@ -33,9 +45,13 @@ exports = module.exports = function (err, description, data) {
 	out.on("open", fd => {
 		out.write(`\n${stack}\n`);
 		out.end();
-	}).on("error", err => {
-		console.error(`\nSUBCRASH: ${err.stack}\n`);
-	});
+	}).on(
+		"error",
+		/** @param {Error} err */
+		err => {
+			console.error(`\nSUBCRASH: ${err.stack}\n`);
+		}
+	);
```

Comment:
Reading this later, I noted that this probably carried the wrong message across.
What I meant was the same as in https://github.com/Zarel/Pokemon-Showdown/pull/3605#discussion_r121272998